prow/tide/tide.go

```diff
@@ -719,20 +719,25 @@ func (c *Controller) pickBatch(sp subpool) ([]PullRequest, error) {
 	if err := r.Checkout(sp.sha); err != nil {
 		return nil, err
 	}
+
+	// we must choose the oldest PRs for the batch
+	sort.Sort(ByNumber(sp.prs))
+
 	var res []PullRequest
-	for i, pr := range sp.prs {
-		// TODO: Make this configurable per subpool.
-		if i == 5 {
-			break
-		}
+	for _, pr := range sp.prs {
 		if !isPassingTests(sp.log, c.ghc, pr) {
 			continue
 		}
 		if ok, err := r.Merge(string(pr.HeadRefOID)); err != nil {
-			return nil, err
+			sp.log.WithFields(pr.logFields()).WithError(err).Warning("Failed to merge.")
```

Comment:
error message should include "merge conflict" ?

Reply:
An error does not indicate a merge conflict: https://github.com/kubernetes/test-infra/blob/a7afc7425839bdc7c9b04f12e1284c9560aa107e/prow/git/git.go#L240-L242

I think we should log this as an error instead of a warning.

Reply:
Oh.  We should comment that the abort failed and stop processing, then.